### PR TITLE
Clean up zookeeper + kafka config

### PR DESCRIPTION
**Pull request by Zapadlo**
Remove redundant sidekicks, now services register themselves in etcd in
ExecStartPost. Moved the etcd location to standard /ft/config

### CODE REVIEW

File: service-files/kafka.service [hunk 1]
```diff
@@ -17,6 +16,10 @@ ExecStart=/bin/bash -c "\
   export IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
   /usr/bin/docker run --rm --name %p -p 9092:9092 --link zookeeper:zk --env \"KAFKA_ADVERTISED_HOST_NAME=$IP\" --env=\"KAFKA_MESSAGE_MAX_BYTES=16777216\" --env=\"KAFKA_REPLICA_FETCH_MAX_BYTES=16777216\" -v /var/run/docker.sock:/var/run/docker.sock wurstmeister/kafka:0.8.2.0"
 ExecStop=/usr/bin/docker stop %p
+ExecStartPost=/bin/sh -c "\
+  KAFKA_IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
+  etcdctl set /ft/config/kafka/ip $KAFKA_IP; \
+  etcdctl set /ft/config/kafka/port 9092:"
```
Comment (catalin-ilea) [inline service-files/kafka.service]: ; instead of :

**Comment by Zapadlo** [comment]
Thanks! Fixed

**Comment by catalin-ilea** [comment]
lgtm

**Comment by Zapadlo** [comment]
Merged into pre-prod and tested there - all good

**Pull request merged**